docs/docs/lists-conditional-rendering.md

```diff
@@ -0,0 +1,264 @@
+---
+id: forms
+title: Lists & Conditional Rendering
+permalink: docs/lists-conditional-rendering.html
+prev: 
+next: 
+---
+
+## Lists
+
+First, let's talk about transforming lists in Javascript.
+
+Given the code below, we use the `map()` function to take an array of `numbers`, double their values, and then output them to a new array `doubled`.
+
+```javascript
+const numbers = [1, 2, 3, 4, 5];
+numbers.map((number) => number * 2); // outputs: [2, 4, 6, 8 10]
+```
+
+In React, transforming arrays into lists of elements is nearly identical.
+
+### Basic List
+
+A component must return only a single item. If you are returning multiple elements, wrap them in `<div></div>`.
+
+Here we take our list of `numbers` and generate a collection of `<li>` elements. We are using the plain Javascript `map()` function. Learn more about it <a target="_blank" href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map">here</a>.
+
+```javascript
+class Numbers extends React.Component {
+  render() {
+    const numbers = [1, 2, 3, 4, 5];
+    return (
+      <div>
+        <ul>
+          {numbers.map((number) => <li>{number}</li>)}
+        </ul>
+      </div>
+    );
+  }
+}
+```
+
+<a target="_blank" href="https://codepen.io/ericnakagawa/pen/wzxEmv/?editors=0011">Try it out on Codepen.</a>
+
```

Comment:
Sorry for duplicate comments. GitHub doesn't show my comments on older threads.

Can we use consistent link format? I'm all for using <a target> for new tab but then we need to fix all other docs and make sure the link text is also the same.